src/controllers/action-queue/WyreClient.ts

```diff
@@ -85,6 +85,7 @@ export const makeWyreClient = async (opt: WyreClientOptions): Promise<WyreClient
       paymentMethodsResponse.forEach(paymentMethod => {
         const { blockchains, id } = paymentMethod
         if (blockchains == null) return
+        if (paymentMethod.status !== 'ACTIVE') return
```

Comment:
It would be better to filter out inactive payment methods at the caller. But for now we will do this.